Source/ACE.Server/Managers/HouseManager.cs

```diff
@@ -405,7 +405,7 @@ private static void HandleRentPaid(PlayerHouse playerHouse)
 
             slumlord.SaveBiotaToDatabase();
 
-            log.Info($"HouseManager.HandleRentPaid({playerHouse.PlayerName}): rent payment successful!");
+            log.Debug($"HouseManager.HandleRentPaid({playerHouse.PlayerName}): rent payment successful!");
```

Comment:
not debug messages.

Reply:
Is this a message that should be printed to the console/server manager when running a server?

I don't think sever operators want to see a live stream of rent being paid.

Reply:
sure, if they want to see a stream of events of certain types that describe what is happening on the server, it's very likely important Housing events such as this would be something they want to see

placing messages into a handful of tiered buckets (if you want to see housing messages, you have to enable debug logging, and everything that comes with it, and then filter it to the messages you DO want to see) seems like it might not be the best system to me. deciding which of 2 buckets messages should go into, or which buckets go to the console, is completely arbitrary. Log(Type, Message) where Type is a Flags enum with all the different messages types (housing, corpse, epics, legendaries, rares) that the server admin can selectively choose to enable/disable independently